.ci/pipeline_definitions

```diff
@@ -55,23 +54,31 @@ etcd-backup-restore:
       unit_test:
         image: 'golang:1.20.3'
       integration_test:
-        image: 'eu.gcr.io/gardener-project/gardener/testmachinery/base-step:stable'
+        image: europe-docker.pkg.dev/gardener-project/releases/testmachinery/base-step:stable
       build:
         image: 'golang:1.20.3'
         output_dir: 'binary'
 
   jobs:
     head-update:
       traits:
-        component_descriptor: ~
         draft_release: ~
+        component_descriptor:
+          ocm_repository_mappings:
+            - repository: europe-docker.pkg.dev/gardener-project/releases
     pull-request:
       traits:
         pull-request: ~
     release:
       traits:
         version:
           preprocess: 'finalize'
+        component_descriptor:
+          ocm_repository: europe-docker.pkg.dev/gardener-project/releases
+        publish:
+          dockerimages:
+            etcdbrctl:
+              image: europe-docker.pkg.dev/gardener-project/releases/gardener/etcdbrctl
```

Comment:
Shouldn't this be `europe-docker.pkg.dev/gardener-project/public/gardener/etcdbrctl`? If not, then [how] are these images copied over? Or is the `public` registry some sort of mirror/proxy for the `releases` + `snapshots` registries?

Reply:
@shreyas-s-rao : `public` is a "virtual" (as GCP calls it) registry that offers a combined view for the union set of `releases` and `snapshots`. so every image in either registry will be visible in `public`, as well.